portfolio.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Shelby Willis - Portfolio</title>
+  <link rel="stylesheet" href="styles/styles.css">
+  <link rel="stylesheet" href="styles/portfolio.css">
+  <link href="https://fonts.googleapis.com/css2?family=Six+Caps&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Yanone+Kaffeesatz:wght@300&display=swap" rel="stylesheet">
+  <script defer src="scripts/header.js"></script>
+</head>
+<body class="bg">
+  <header id="header" class="navlinksmain"></header>
+  <section class="container">
+  <a class="box" href="https://github.com/TiaWillis1/viewing-party" aria-label="Viewing Party" 
+  target="_blank">
+  <section>
+    <header class="projects">
+      <h3>
+        <div>Viewing Party</div>
+      </h3>
+    </header>
+    <p>
+      Viewing Party is a project that helps friends connect over movies by tracking 
+      and organzing watchlists while also making new movie suggestions!
+    </p>
+  </section>
```

Comment:
Let's turn these into `section` or `article` elements, and then let th `h3` stand on its own
```html
  <section>
    <h3 class="projects">Viewing Party</h3>
    <p>
      Viewing Party is a project that helps friends connect over movies by tracking 
      and organzing watchlists while also making new movie suggestions!
    </p>
  </section>
```